sockeye/data_io.py

```diff
@@ -1698,12 +1698,12 @@ def iter_next(self) -> bool:
             source_len = 0 if sources[0] is None else len(sources[0])
             target_len = 0 if target is None else len(target)
             if source_len > self.max_len_source:
-                logger.info("Trimming source sentence {} ({} -> {})".format(self.sentno + num_read,
+                logger.debug("Trimming source sentence {} ({} -> {})".format(self.sentno + num_read,
                                                                             source_len,
```

Comment:
nit: the argument is no longer aligned.

Personally I would continue to show this message to the user and not convert to debug.